src/UglyToad.PdfPig/DocumentLayoutAnalysis/MathExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace UglyToad.PdfPig.DocumentLayoutAnalysis
+{
+    /// <summary>
+    /// Useful math extensions.
+    /// </summary>
+    public static class MathExtensions
+    {
+        /// <summary>
+        /// Computes the mode of a sequence of float values.
+        /// </summary>
+        /// <param name="array"></param>
```

Comment:
Just some empty doc-comments to purge in here, the `Mode` enumerates twice but I'm ok with it for the time being since it's a bit of pain to unroll.